run-dask-process.sh

```diff
@@ -106,6 +105,8 @@ function buildTcpArgs {
     WORKER_ARGS="--rmm-pool-size=$WORKER_RMM_POOL_SIZE
              --local-directory=/tmp/$LOGNAME 
              --scheduler-file=$SCHEDULER_FILE
+             --memory-limit=$DASK_HOST_MEMORY_LIMIT
```

Comment:
Nice you added these arguments as options. I played with them when trying to catch an illegal memory access followed by an OOM for some MNMG algos like WCC